python/lsst/pipe/supertask/activator.py

```diff
@@ -392,6 +428,10 @@ def parse_and_run(cls):
                                       help="Manually adds packages to look for task, overrides "
                                            "TASK_PACKAGES, to see the available packages use -lp "
                                            "option")
+        parser_activator.add_argument('--package_paths', nargs='+', default=(),
+                                      help="Manually adds path of packages to look for task, overrides "
+                                           "TASK_PACKAGES, to see the available packages use -lp "
+                                           "option")
```

Comment:
It can be full or relative path w/r the CWD.
The path should be pointing to WHERE the *.py is, not to the file itself, in case more paths are added or there more than one module in the same location. This is subtle.  So if you want to run a task sitting inside a file in your $CWD, you would do:
cmdLineActivator  MyTask --package_paths . --extras ....  OR
cmdLineActivator --package_paths $PWD --extras ....